diff --git a/luci-app-adguardhome/Makefile b/luci-app-adguardhome/Makefile
index db03e8a..5c71061 100644
--- a/luci-app-adguardhome/Makefile
+++ b/luci-app-adguardhome/Makefile
@@ -16,7 +16,7 @@ LUCI_DESCRIPTION:=LuCI support for AdGuardHome
 define Package/$(PKG_NAME)/config
 config PACKAGE_$(PKG_NAME)_INCLUDE_binary
 	bool "Include Binary File"
-	default y
+	default n
 endef
 
 PKG_CONFIG_DEPENDS:= CONFIG_PACKAGE_$(PKG_NAME)_INCLUDE_binary
@@ -24,7 +24,7 @@ PKG_CONFIG_DEPENDS:= CONFIG_PACKAGE_$(PKG_NAME)_INCLUDE_binary
 define Package/luci-app-adguardhome/conffiles
 /usr/share/AdGuardHome/links.txt
 /etc/config/AdGuardHome
-/etc/AdGuardHome.yaml
+#/etc/AdGuardHome.yaml
 endef
 
 define Package/luci-app-adguardhome/postinst
diff --git a/luci-app-adguardhome/root/etc/config/AdGuardHome b/luci-app-adguardhome/root/etc/config/AdGuardHome
index 6ecfde8..4136c0a 100644
--- a/luci-app-adguardhome/root/etc/config/AdGuardHome
+++ b/luci-app-adguardhome/root/etc/config/AdGuardHome
@@ -2,7 +2,7 @@ config AdGuardHome 'AdGuardHome'
 	option enabled '0'
 	option httpport '3000'
 	option redirect 'none'
-	option configpath '/etc/AdGuardHome.yaml'
+	option configpath '/etc/config/AdGuardHome.yaml'
 	option workdir '/etc/AdGuardHome'
 	option logfile '/tmp/AdGuardHome.log'
 	option verbose '0'
