diff --git a/luci-app-cpufreq/Makefile b/luci-app-cpufreq/Makefile
index 5a95b05..ab0b852 100644
--- a/luci-app-cpufreq/Makefile
+++ b/luci-app-cpufreq/Makefile
@@ -5,7 +5,7 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=LuCI for CPU Freq Setting
-LUCI_DEPENDS:=@(arm||aarch64)
+#LUCI_DEPENDS:=@(arm||aarch64)
 
 PKG_NAME:=luci-app-cpufreq
 PKG_VERSION:=1
diff --git a/luci-app-cpufreq/root/etc/config/cpufreq b/luci-app-cpufreq/root/etc/config/cpufreq
index 5c2c070..c61ab82 100644
--- a/luci-app-cpufreq/root/etc/config/cpufreq
+++ b/luci-app-cpufreq/root/etc/config/cpufreq
@@ -1,3 +1,13 @@
 
 config settings 'cpufreq'
-
+	option governor0 'schedutil'
+	option maxfreq0 '1608000'
+	option governor4 'schedutil'
+	option maxfreq4 '2016000'
+	option minfreq0 '600000'
+	option minfreq4 '600000'
+	option maxfreq '716000'
+	option upthreshold '50'
+	option factor '10'
+	option minifreq '300000'
+	option governor 'ondemand'
\ No newline at end of file
diff --git a/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq b/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq
index 0bdab1a..36e1ac6 100755
--- a/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq
+++ b/luci-app-cpufreq/root/etc/uci-defaults/10-cpufreq
@@ -62,11 +62,13 @@ case "$DISTRIB_TARGET" in
 	"rockchip/armv8")
 		if echo "$CPU_POLICYS" | grep -q "4"; then
 			# RK3399
-			uci_write_config 0 schedutil 600000 1608000
-			uci_write_config 4 schedutil 600000 2016000
+			uci_write_config 0 schedutil 600000 1800000
+			uci_write_config 4 schedutil 600000 2208000
+		elif [[ "$(board_name)" == *nanopi-r5s ]]; then
+			uci_write_config 0 schedutil 816000 1992000
 		else
 			# RK3328
-			uci_write_config 0 schedutil 816000 1512000
+			uci_write_config 0 schedutil 816000 1608000
 		fi
 		;;
 	"sunxi/cortexa53")
